### PR TITLE
containerapp: fix create or update containerapp --yaml exposedPort

### PR DESCRIPTION
**Pull request by Greedygre**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command

az containerapp env update
az containerapp create/update --yaml

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
containerapp

**Review by howang-ms — APPROVED** [review]

File: src/containerapp/azext_containerapp/_sdk_models.py [hunk 1]
```diff
@@ -2096,23 +2106,29 @@ class Ingress(Model):
...
         self.fqdn = None
         self.external = kwargs.get('external', False)
+        self.exposedPort = kwargs.get('exposed_port', None)
         self.target_port = kwargs.get('target_port', None)
         self.transport = kwargs.get('transport', None)
         self.traffic = kwargs.get('traffic', None)
         self.custom_domains = kwargs.get('custom_domains', None)
-        self.allow_insecure = kwargs.get('allow_insecure', None)
-        self.ipSecurityRestrictions = kwargs.get('ipSecurityRestrictions', None)
+        self.allow_insecure = kwargs.get('allow_insecure', False)
```
Comment (Juliehzl) [inline src/containerapp/azext_containerapp/_sdk_models.py:2128]: behavior change?

File: src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py [hunk 2]
```diff
@@ -936,6 +936,136 @@ def test_containerapp_create_with_yaml(self, resource_group):
...
+                    tags:
+                        tagname: value
+                    properties:
+                      environmentId: {containerapp_env["id"]}
+                      configuration:
+                        activeRevisionsMode: Multiple
+                        ingress:
+                          external: true
+                          allowInsecure: false
+                          targetPort: 80
```
Comment (Juliehzl) [inline src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py:1006]: test with exposedPort

Comment (Greedygre) [reply src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py:1006]: exposedPort only support for TCP transport. Test exposedPort in test_containerapp_create_with_vnet_yaml.

File: src/containerapp/azext_containerapp/_sdk_models.py (hunk 1 above)
Comment (Greedygre) [reply src/containerapp/azext_containerapp/_sdk_models.py:2128]: In SDK the default value is false.
Should I follow SDK default value?

File: src/containerapp/HISTORY.rst [hunk 3]
```diff
@@ -4,6 +4,7 @@ Release History
 ===============
 Upcoming
 +++++++
+* 'az containerapp create/update': --yaml support models for api-version 2022-10-01
```
Comment (Juliehzl) [inline src/containerapp/HISTORY.rst:7]: add supported properties

Comment (Greedygre) [reply src/containerapp/HISTORY.rst]: Done

**Comment by Greedygre** [comment]
@zhoxing-ms 
 Hi, this PR has been reviewed by hong and zunli, and it needs to be merged before https://github.com/Azure/azure-cli-extensions/pull/6081
 please help to review and merge, thanks!

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**